book.rb

```diff
@@ -0,0 +1,27 @@
+# class Book
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book, person, date)
```

Comment:
- [ ] You only need to add your person, and date. And rest of the features need to be handled in the Rental class. Kindly remove `book` from add_rental params.